libsolidity/interface/SourceReferenceFormatter.cpp

```diff
@@ -64,72 +60,67 @@ void SourceReferenceFormatter::printSourceLocation(
 			endColumn = startColumn + locationLength;
 		}
 
-		_stream << line << endl;
+		m_stream << line << endl;
+
 		for_each(
 			line.cbegin(),
 			line.cbegin() + startColumn,
-			[&_stream](char const& ch) { _stream << (ch == '\t' ? '\t' : ' '); }
+			[this](char const& ch) { m_stream << (ch == '\t' ? '\t' : ' '); }
```

Comment:
Not sure which is faster for closures, giving the entire context or just the member variable?

Reply:
It's not a big object for copy, so I don't think it makes that much of a difference here.